tests/integration_tests/test_cli.py

```diff
@@ -41,6 +41,13 @@ def test_bad_config_error_message(tmp_path):
         run_cli(TEST_RUN_MODE, "--disable-monitor", str(tmp_path / "test.ert"))
 
 
+def test_test_run_on_lsf_configuration_works_with_no_errors(tmp_path):
+    (tmp_path / "test.ert").write_text(
+        "NUM_REALIZATIONS 1\nQUEUE_SYSTEM LSF", encoding="utf-8"
```

Comment:
Maybe just add one `QUEUE_OPTION LSF PROJECT PROJECT` to make sure we populate some incompatible options.

Reply:
The options were not really part of the problem in this bug. There are other tests asserting which errors or warnings arise when incompatible options are supplied, I think they suffice.